Functions.Templates/Templates-v2/BlobTrigger-JavaScript/template.json

```diff
@@ -0,0 +1,47 @@
+{
+    "author": "Eric Jizba",
+    "name": "Azure Blob Storage trigger",
+    "description": "$BlobTrigger_description",
+    "programmingModel": "v4",
+    "language": "JavaScript",
+    "jobs": [
+        {
+            "name": "Create New Function",
+            "type": "CreateNewFile",
+            "inputs": [
+                {
+                    "assignTo": "$(FUNCTION_NAME_INPUT)",
+                    "paramId": "functionName",

```

Comment:
I don't see a `functionName` in [`userPrompts.json`](https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Bindings/userPrompts.json). Perhaps you meant `trigger-functionName`?